src/parquet/column/scanner.h

```diff
@@ -62,14 +64,14 @@ class Scanner {
     return reader_->descr();
   }
 
-  size_t batch_size() const { return batch_size_;}
+  int32_t batch_size() const { return batch_size_;}
 
-  void SetBatchSize(size_t batch_size) {
+  void SetBatchSize(int32_t batch_size) {
     batch_size_ = batch_size;
   }
 
  protected:
-  size_t batch_size_;
+  int32_t batch_size_;
```

Comment:
Will make `int64_t` -- data pages are limited to `int32_t` but a row group could in theory be larger.